kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -467,14 +479,14 @@ def pretty(self, *, details: bool = True) -> str:
         return '\n'.join(lines)
 
 
-def foundry_list(foundry_out: Path) -> List[CfgStat]:
-    kcfgs_dir = foundry_out / 'kcfgs'
+def foundry_list(foundry_root: Path) -> List[CfgStat]:
+    kcfgs_dir = get_foundry_out_from_root(foundry_root) / 'kcfgs'
     paths = kcfgs_dir.glob('*.json')
     return [CfgStat.from_file(path) for path in paths]
```

Comment:
```suggestion
def foundry_list(foundry_root: Path) -> List[CfgStat]:
    foundry = Foundry(foundry_root)
    kcfgs_dir = foundry.out / 'kcfgs'
    paths = kcfgs_dir.glob('*.json')
    return [CfgStat.from_file(path) for path in paths]
```

With `out` declared as a `property` of `Foundry` class, we can use it like this.